docs/linked_services.md

```diff
@@ -0,0 +1,21 @@
+# Linked Services
+
+## Forms Engine Configuration
```

Comment:
Does the forms config apply to the store?

Reply:
no that file shouldnt be there, removed :)